nixos/modules/tasks/filesystems/bcachefs.nix

```diff
@@ -99,8 +99,7 @@ in
       system.fsPackages = [ pkgs.bcachefs-tools ];
 
       # use kernel package with bcachefs support until it's in mainline
```

Comment:
It's already in mainline, you should replace the comment with something along the lines of
```suggestion
# Linux 6.7 is required for bcachefs support
# FIXME: replace this with linuxPackages_latest when 6.7 hits stable, remove this line when the LTS version is at least 6.7
```